src/workerd/api/worker-rpc.c++

```diff
@@ -1930,6 +1930,7 @@ kj::Promise<WorkerInterface::CustomEvent::Result> JsRpcSessionCustomEventImpl::r
     // throwing, rather than some generic revocation exception.
     auto e = kj::getCaughtExceptionAsKj();
     revcableTarget.revoke(kj::cp(e));
+    incomingRequest->finishScheduled();
```

Comment:
Not sure what the goal is here, but note that there is already a `KJ_DEFER` block above which runs `drain()` -- and defer blocks apply even on exceptions. It's incorrect to run both `drain()` and `finishScheduled()`.